packages/react-experiments/src/components/UnifiedPicker/UnifiedPicker.tsx

```diff
@@ -66,6 +66,16 @@ export const UnifiedPicker = <T extends {}>(props: IUnifiedPickerProps<T>): JSX.
     onInputChange,
   } = props;
 
+  const defaultDragDropEnabled = React.useMemo(
+    () => (props.defaultDragDropEnabled !== undefined ? props.defaultDragDropEnabled : true),
```

Comment:
can props.defaultDragDropEnabled be null too ? i would check on !props.defaultDragDropEnabled

Reply:
I tried that - if the prop is set, it then does a bool compare, so this wasn't working. I should check for null too though, I'll do some comparisons to see what statement will work

Reply:
Though also can you set a boolean value to null?

Reply:
Ok I checked, you can't, this value will be either undefined or a boolean